connPFM/cli/connPFM.py

```diff
@@ -59,13 +59,13 @@ def _get_parser():
         "-te",
         "--te",
         dest="te",
-        type=list,
```

Comment:
Why don't we get this from the number of input files?

Reply:
What do you mean? We need the TE values to generate the HRF convolution matrix in multi-echo.